### ISSUE TITLE
Implement validation

### ISSUE THREAD

**Issue by s-broda** [s-broda/nmt#6]
After every few epochs, we should probably do validation. The validation set is already loaded in train_transformer; just need to evaluate the BLEU score on it. Maybe not the whole thing though, because that might take more than an hour, which is in the same ballpart as training for one epoch.
